package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@inlet/react-pixi": "^0.3.0",
```

Comment:
This seems like the proper module to use when we want Pixi.js and React at the same time.

Reply:
Never thought of using it in this way. Could be an interesting workflow. 👍